### PR TITLE
This is my first test with GitHub

### PR DESCRIPTION
**Pull request by htarrido-cuny**
I am teaching my students how github works. Adding some dummy files that can be deleted, maybe, later.

### CODE REVIEW

**Comment by Arc-Aly** [comment]
Line 36 of your code is wrong!

**Comment by Tansunur50** [comment]
Hey Hector, your code is great!

**Comment by htarrido-cuny** [comment]
> Line 36 of your code is wrong!

Hey @Arc-Aly which file??????

File: Workshops/InDesign/Git.rtf [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{\pard \s1 \f1\fs32\ql\sb240\sa180\b {\*\bkmkstart 3.workingonyourlocalbranch}{\*\bkmkend 3.workingonyourlocalbranch}3. Working on your local branch\par}
+
+{\pard \s2 \f1\fs28\ql\sb240\sa180\i\b {\*\bkmkstart gitstatus}{\*\bkmkend gitstatus}git status\par}
+
+{\pard \s2 \f1\fs28\ql\sb240\sa180\i\b {\*\bkmkstart gitadd}{\*\bkmkend gitadd}git add\par}
+
+{\pard \s2 \f1\fs28\ql\sb240\sa180\i\b {\*\bkmkstart gitcommit-m}{\*\bkmkend gitcommit-m}git commit -m\par}
+
+{\pard \s1 \f1\fs32\ql\sb240\sa180\b {\*\bkmkstart 4.pushingreviewingandmerging}{\*\bkmkend 4.pushingreviewingandmerging}4. Pushing, reviewing and merging\par}
+
```
Comment (htarrido-cuny) [inline Workshops/InDesign/Git.rtf:36]: Hey hector this it the file I was talking about. Please fix this.

**Review by htarrido-cuny — COMMENTED** [review]
test

**Comment by htarrido-cuny** [comment]
# Example
## Example
### Example

_italize_

**Comment by Tansunur50** [comment]
#EXAMPLE
##EXAMPLE

_italize_

**Pull request merged**